.travis.yml

```diff
@@ -7,14 +7,24 @@ python:
     - 3.6
     - pypy
     - pypy3
+matrix:
+    include:
+        - python: "3.7"
+          dist: xenial
+          sudo: true
 install:
+    - pip install zope.testrunner coverage coveralls
     - pip install zc.buildout
+    - pip install -U -e .[test]
     - buildout
```

Comment:
Do we still need zc.buildout?  AFAICS it's only used to install sphinx (and repoze.sphinx.autointerface) now.

Reply:
+1 to dropping buildout from .travis.yml. tox.ini doesn't use it (tox.ini needs its own set of updates.) Maybe that's a separate PR.